docs/advanced-backtesting.md

```diff
@@ -114,3 +115,37 @@ For example, if your backtest timerange was `20220101-20221231` but you only wan
 ```bash
 freqtrade backtesting-analysis -c <config.json> --timerange 20220101-20220201
 ```
+
+### Printing out rejected trades
+
+Use the `--rejected` option to print out rejected trades.
```

Comment:
You will clearly have to explain what "rejected" trades are.

to me - these are trade that 
* got rejected by the user via confirm_trade_entry
* got rejected by the market - by not filling the order (so the initial entry is canceled).

I think you mean neither of these ... so the term rejected is probably wrong.

Reply:
I'll commit this later.